Resources/MixedModels/qml/common/MixedModelsPlots.qml

```diff
@@ -171,7 +173,7 @@ Section
 			visible:		plotsGeom.currentValue == "geom_jitter" | plotsGeom.currentValue == "geom_boxjitter"
```

Comment:
A single pipe is [bitwise or](https://stackoverflow.com/questions/6194950/what-does-the-single-pipe-do-in-javascript)
```suggestion
			visible:		plotsGeom.currentValue == "geom_jitter" || plotsGeom.currentValue == "geom_boxjitter"
```